module/src/impls/process.rs

```diff
@@ -0,0 +1,250 @@
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+use crate::link::{self, Linkable, Linker, Port};
+use crate::sandbox::{self, Sandbox, Sandboxer};
+use crossbeam::thread;
+use fproc_sndbx::execution::{executor, with_rto};
+use fproc_sndbx::ipc::Ipc;
+use parking_lot::Mutex;
+use remote_trait_object::HandleToExchange;
+use std::io::Cursor;
+use std::marker::PhantomData;
+use std::path::Path;
+use std::sync::Arc;
+
+/// ProcessSandboxer is really trivial, because there is nothing really to do
+/// for the processes. It just creates ProcessSandbox by demand, and let it just be.
+/// It doesn't even hold its sandboxes.
+pub struct ProcessSandboxer<E: ExecutionScheme> {
+    _p: PhantomData<E>,
+}
+
+impl<E: ExecutionScheme> Sandboxer for ProcessSandboxer<E> {
+    fn id(&self) -> &'static str {
+        unimplemented!()
+    }
+
+    fn supported_module_types(&self) -> &'static [&'static str] {
+        unimplemented!()
+    }
+
+    fn load<'a>(
+        &self,
+        path: &'a dyn AsRef<Path>,
+        init: &dyn erased_serde::Serialize,
+        exports: &[(&str, &dyn erased_serde::Serialize)],
+    ) -> Result<Box<dyn Sandbox>, sandbox::Error<'a>> {
+        let mut init_buffer = Vec::<u8>::new();
+        let cbor = &mut serde_cbor::Serializer::new(serde_cbor::ser::IoWrite::new(Cursor::new(&mut init_buffer)));
+        init.erased_serialize(&mut erased_serde::Serializer::erase(cbor)).unwrap();
+
+        let exports: Vec<(String, Vec<u8>)> = exports
+            .iter()
+            .map(|(name, data)| {
+                let mut buffer = Vec::<u8>::new();
+                let cbor = &mut serde_cbor::Serializer::new(serde_cbor::ser::IoWrite::new(Cursor::new(&mut buffer)));
+                data.erased_serialize(&mut erased_serde::Serializer::erase(cbor)).unwrap();
+                (name.to_string(), buffer)
+            })
+            .collect();
+
+        Ok(Box::new(ProcessSandbox::<E>::new(path.as_ref(), &init_buffer, &exports)?))
+    }
+}
+
+impl<E: ExecutionScheme> ProcessSandboxer<E> {
+    #[allow(clippy::new_without_default)]
+    pub fn new() -> Self {
+        Self {
+            _p: PhantomData,
+        }
+    }
+}
+
+pub trait ExecutionScheme: Send + Sync + 'static {
+    type Ipc: fproc_sndbx::ipc::Ipc + 'static;
+    type Execution: executor::Executor;
+    fn is_intra() -> bool;
+}
+
+pub struct MultiProcess;
+impl ExecutionScheme for MultiProcess {
+    type Ipc = fproc_sndbx::ipc::unix_socket::DomainSocket;
+    type Execution = fproc_sndbx::execution::executor::Executable;
+    fn is_intra() -> bool {
+        false
+    }
+}
+
+pub struct SingleProcess;
+impl ExecutionScheme for SingleProcess {
+    type Ipc = fproc_sndbx::ipc::intra::Intra;
+    type Execution = fproc_sndbx::execution::executor::PlainThread;
+    fn is_intra() -> bool {
+        true
+    }
+}
+
+#[remote_trait_object_macro::service]
+pub trait SandboxForModule: remote_trait_object::Service {
+    fn ping(&self);
+}
```

Comment:
Do you have any plan to add functions in the `SandboxForModule`?
If not, we don't have to use rto context between sandboxee and sandboxer.